### PR TITLE
Common questions

### PR DESCRIPTION
**Pull request by alvloureiro**
Add FAQ page implementation
Remove question related to git commands from git-commands.html page

### CODE REVIEW

File: src/faq.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<html>
+    <body>
+        <h1>Git Frequently Asked Questions</h1>
+        <p>
+            <a href="./index.html">Back</a>
+        </p>
+
+        <h2>How to create a local git repository?</h2>
+        <p>
+            $ git init .
```
Comment (alvloureiro) [inline src/faq.html:10]: Add strong tag between command linke insertion

File: src/faq.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        </p>
+
+        <h2>How to create a branch</h2>
+        <p>
+            $ git branch <branch-name>
+        </p>
+    </body>
+</html>
```
Comment (Araunhoz) [inline src/faq.html:18]: Need a blank line in the end of file

**Review by alvloureiro — COMMENTED** [review]
Need Rework

File: src/faq.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        </p>
+
+        <h2>How to create a local git repository?</h2>
+        <p>
+            $ git init .
+        </p>
+
+        <h2>How to create a branch</h2>
+        <p>
+            $ git branch <branch-name>
```
Comment (Araunhoz) [inline src/faq.html:15]: Consider fixing the Branch-name tag

**Review by Araunhoz — CHANGES_REQUESTED** [review]
Need changes on the first file

**Review by Araunhoz — APPROVED** [review]
LGTM

**Review by Araunhoz — APPROVED** [review]
LGTM

**Review by MailtonViana — APPROVED** [review]

**Review by Araunhoz — APPROVED** [review]
LGTM

**Pull request merged**